frontend/src/components/messstelle/optionsmenue/panels/content/FahrzeugPanelVerkehrsartenContent.vue

```diff
@@ -165,6 +165,13 @@
         </v-row>
       </v-col>
       <v-col cols="4">
+        <p
+          v-if="!hasSelectedVerkehrsarten"
+          class="text-red"
+        >
+          Wenn keine Verkehrsart ausgewählt wird, kann der Belastungsplan nicht
```

Comment:
Evtl ändern bzw. neutraler Formulieren:

Beispiel: "Bei fehlender Auswahl der Verkehrsart ist eine anzeige des Belastungsplan nicht mölgich."

Den Text mit dem oberen Text abstimmen: https://github.com/it-at-m/dave-frontend/pull/177/files#diff-a841a6dbe6ee857ba57dada767d8f89db309decc8715957318e00dc0f6b9bea3R50